models/tasks.js

```diff
@@ -341,6 +341,26 @@ const overdueTasks = async (overDueTasks) => {
     throw err;
   }
 };
+
+/**
+ * Fetch all overdue tasks objects
+ * @param overdueTasks <Array>: tasks which are overdue
+ * @return {Promsie<Array>}
+ */
+const getAllOverDueTasks = async () => {
+  try {
+    const currentTime = Math.floor(Date.now() / 1000);
+    const overdueTasksSnapshot = await tasksModel.where("endsOn", "<", currentTime).get();
+    const tasks = buildTasks(overdueTasksSnapshot);
+    const promises = tasks.map((task) => fromFirestoreData(task));
```

Comment:
I guess we can name this to something like `overdueTasksPromises`...something that is more semantically sound.

Reply:
+1